crates/aptos/src/move_tool/mod.rs

```diff
@@ -922,6 +924,24 @@ pub(crate) enum FunctionArgType {
     U64,
     U128,
     Raw,
+    Vector(Box<FunctionArgType>),
+}
+
+impl Display for FunctionArgType {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        match self {
+            FunctionArgType::Address => write!(f, "address"),
+            FunctionArgType::Bool => write!(f, "bool"),
+            FunctionArgType::Hex => write!(f, "hex"),
+            FunctionArgType::HexArray => write!(f, "hex_array"),
+            FunctionArgType::String => write!(f, "string"),
+            FunctionArgType::U8 => write!(f, "u8"),
```

Comment:
Just to confirm, we're not missing u16 or u32 are we? Similarly do we support u8 and u128 yet? Trying to remember what the changes are that Victor wanted to make...